opencog/atoms/execution/ExecutionOutputLink.cc

```diff
@@ -105,14 +101,19 @@ ExecutionOutputLink::ExecutionOutputLink(const Link& l)
 ///
 Handle ExecutionOutputLink::execute(AtomSpace* as, bool silent) const
 {
+	if (_outgoing[0]->getType() != GROUNDED_SCHEMA_NODE) {
+		LAZY_LOG_FINE << "Not a grounded schema. Do not execute it";
```

Comment:
I think it's because "unfolding" defined schemata occurs earlier in the process, that's the idea I was left with after reviewing `do_execute`. Since all unit tests pass I assumed it was OK.